event-handlers/src/test/java/no/sikt/nva/nvi/events/evaluator/dto/ContributorDtoTest.java

```diff
@@ -20,7 +20,6 @@ class ContributorDtoTest {
               "verificationStatus": "NotVerified",
               "additionalIdentifiers": []
           },
-          "affiliations": [],
```

Comment:
This is to provoke the error? We could do destruction testing on stupid cases directly with a null and empty source in a parameterized test

Reply:
Indeed, just a lazy way to test the actual scenario I had. Should parameterize it to test both cases.